pkg/cvo/metrics.go

```diff
@@ -243,13 +243,14 @@ func (m *operatorMetrics) Describe(ch chan<- *prometheus.Desc) {
 	ch <- m.clusterOperatorUp.WithLabelValues("", "").Desc()
 	ch <- m.clusterOperatorConditions.WithLabelValues("", "", "").Desc()
 	ch <- m.clusterOperatorConditionTransitions.WithLabelValues("", "").Desc()
-	ch <- m.clusterInstaller.WithLabelValues("", "", "").Desc()
+	ch <- m.clusterInstaller.WithLabelValues("", "", "", "").Desc()
 	ch <- m.clusterVersionOperatorUpdateRetrievalTimestampSeconds.WithLabelValues("").Desc()
 }
 
 func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 	current := m.optr.currentVersion()
 	var completed configv1.UpdateHistory
+	clusterId := "<missing>"
```

Comment:
Generally we would never do this.  clusterID cannot be missing except when data is unavailable, so empty string is appropriate.

Reply:
Okay, Thank you @smarterclayton  I wasn't sure. Will remove this one.

Reply:
Changed to "", I guess I still may need some advice with Test Unit. Do you know who could help me out? It was a bit confusing at first, I understand it bit better now. But unsure what scenarios needs to be tested with cluster_id. I added the same where we test cluster-installer type